app/views/root_taxons/_bubbles.html.erb

```diff
@@ -0,0 +1,60 @@
+<style>
```

Comment:
there are a few inline styles in this file, which should probably be pulled out into their own stylesheet and invoked using class names and inheritance.

Reply:
I've done this in 7ddb773